design/src/main/res/values/colors.xml

```diff
@@ -16,6 +16,9 @@
     <color name="mercury">#7A8085</color>
     <color name="mercury_light">#989FA3</color>
     <color name="mercury_crystal">#DDDFE0</color>
+    <color name="mercury_70">#44494d</color>
```

Comment:
These names are from DS project?

Reply:
Use uppercased letters for color Hex strings